src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 10 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

PSModulePath changes interacting with other PowerShell processes.System32 module path prepended to PSModulePath.Allows PowerShell subprocesses to call core modules
```powershell
Expected $null, but got @(Get-ChildItem : The 'Get-ChildItem' command was found in the module 'Microsoft.PowerShell.Management', but the module could not be loaded. For more information, run 'Import-Module Microsoft.PowerShell.Management'., At line:1 char:1, + Get-ChildItem, + ~~~~~~~~~~~~~, + CategoryInfo          : ObjectNotFound: (Get-ChildItem:String) [], CommandNotFoundException, + FullyQualifiedErrorId : CouldNotAutoloadMatchingModule,  ).
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 353
353:             $errors | Should -Be $null
```
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```
ForEach-Object -Parallel common parameters.Verifies that ErrorAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:128 char:40
    + ... k = { 1..1 | ForEach-Object -Parallel { "Hello" } -ErrorAction Stop }
    +                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that WarningAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:132 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -WarningAction Silen ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that InformationAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:136 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -InformationAction S ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```